ui/src/tasks/actions/index.ts

```diff
@@ -558,12 +560,15 @@ export const createTaskFromTemplate = (template: ITaskTemplate) => async (
 export const runDuration = (finishedAt: Date, startedAt: Date): string => {
   let timeTag = 'seconds'
 
-  let diff = (finishedAt.getTime() - startedAt.getTime()) / 1000
+  if (!isNaN(finishedAt.getTime()) || !isNaN(startedAt.getTime())) {
```

Comment:
so I think this statement will still be true if one is nan but one isn't.  and should be (!isNan(...) && !isNan(...)) and instead of or.

Reply:
But actually I suggest doing
```
if (isNaN(finishedAt.getTime()) || isNaN(startedAt.getTime())){
return '' 
}
// rest of logic
```

Reply:
Will update!